pulsar-broker-auth-sasl/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderSasl.java

```diff
@@ -175,6 +186,24 @@ private String createAuthRoleToken(String role, String sessionId) {
         return signed;
     }
 
+    private byte[] readSecretFromUrl(String secretConfUrl) throws IOException {
+        if (secretConfUrl.startsWith("file:")) {
+            try {
+                return IOUtils.toByteArray(URL.createURL(secretConfUrl.trim()));
+            } catch (IOException e) {
+                throw e;
+            } catch (Exception e) {
+                throw new IOException(e);
+            }
```

Comment:
I think this exception handling could be omitted. Catching all exceptions isn't really justified in this case.

Reply:
yes , I will remove.